### PR TITLE
feat: upgrade via pricing page

### PR DESCRIPTION
**Pull request by alaister**
## What kind of change does this PR introduce?

feature

## What is the current behavior?

Users will often accidentally create new paid orgs when they meant to upgrade an exisiting org.

## What is the new behavior?

If a user is logged in, the buttons on the pricing page will show an "Upgrade now" button instead of "Get started". They will then be able to select an existing org they wish to upgrade and be redirected to the relevant page in the dashboard.

### CODE REVIEW

File: apps/www/lib/fetchWrapper.ts [hunk 1]
```diff
@@ -1,18 +1,46 @@
...
+    headers.set('Authorization', `Bearer ${accessToken}`)
+  }
+
   return fetch(url, {
     method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-      Accept: 'application/json',
-    },
+    headers: headers,
```
Comment (kevcodez) [inline apps/www/lib/fetchWrapper.ts:41]:
```suggestion
    headers,
```

File: apps/www/lib/fetchWrapper.ts [hunk 2]
```diff
@@ -1,18 +1,46 @@
...
+    Accept: 'application/json',
+  })
+
+  if (accessToken) {
+    headers.set('Authorization', `Bearer ${accessToken}`)
+  }
+
+  return fetch(url, {
+    method: 'GET',
+    headers: headers,
```
Comment (kevcodez) [inline apps/www/lib/fetchWrapper.ts:21]:
```suggestion
    headers,
```

**Review by kevcodez — APPROVED** [review]
Tested on staging

File: apps/www/components/Pricing/PricingComparisonTable.tsx [hunk 3]
```diff
@@ -327,54 +362,70 @@ const PricingComparisonTable = () => {
...
+                          )}
+                        </p>
+                      </span>
+                      <span className="flex flex-col justify-between h-full pb-2">
+                        {isUpgradablePlan && hasExistingOrganizations ? (
+                          <UpgradePlan
+                            organizations={organizations}
+                            onClick={() =>
+                              sendTelemetryEvent(
+                                gaEvents[`www_pricing_comparison_${plan.name.toLowerCase()}_mobile`]
```
Comment (fsansalvadore) [inline apps/www/components/Pricing/PricingComparisonTable.tsx:400]: This should be "not" `_mobile`, like line 416.
The "Upgrade plan"s could have their own separate event btw.

File: apps/www/components/Pricing/UpgradePlan.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                  </CommandGroup_Shadcn_>
+                </CommandList_Shadcn_>
+              </Command_Shadcn_>
+            </PopoverContent_Shadcn_>
+          </Popover_Shadcn_>
+        </DialogSection>
+
+        <DialogSection>
+          <DialogDescription className="text-xs">
+            Upon continuing, you will be redirected to the organization&apos;s billing page where
```
Comment (fsansalvadore) [inline apps/www/components/Pricing/UpgradePlan.tsx:132]: Maybe `By continuing you will...`?

**Review by fsansalvadore — APPROVED** [review]
Left two comments but approving to unblock.

File: apps/www/components/Pricing/PricingComparisonTable.tsx (hunk 3 above)
Comment (alaister) [reply apps/www/components/Pricing/PricingComparisonTable.tsx]: Nice catch! Removed `_mobile` there and added a new event for the upgrade buttons :)

**Pull request merged**